html/canvas/offscreen/manual/draw-generic-family/2d.text.draw.generic.family.w.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<title>OffscreenCanvas test: 2d.text.draw.generic.family.w</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/html/canvas/resources/canvas-tests.js"></script>
+<script id='myWorker' type='text/worker'>
+self.onmessage = function(e) {
+  var getOffscreenCanvasForFontShorthand = function(fontShorthand) {
+    var oc = new OffscreenCanvas(300, 150);
+    var ctx = oc.getContext('2d');
+    ctx.font = fontShorthand;
+    ctx.fillText(fontShorthand, 0, 32);
+    return oc;
+  };
+
+  var shorthands = e.data.shorthands;
+  var ret = [];
+  for (var i = 0; i < shorthands.length; ++i) {
+     var oc = getOffscreenCanvasForFontShorthand(shorthands[i]);
+     var imageBitmap = oc.transferToImageBitmap();
```

Comment:
Why not use getImageData here and transfer the result of that? A comment would help.

Reply:
I didn't initially because the result of getImageData isn't transferrable - but I assume it could be copied. This seems to be a pattern used in some other tests, but I don't have strong feelings about it either way.